CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.17.0
+
+- Added support for escape codes in identifiers.
```

Comment:
Is this a breaking change? Does it have to be?

Reply:
In practice no, but I encountered a very peculiar edge case to which this is breaking.

Previously the parser would accept (note the space):
```
# id {
}
```
and treat it as
```
#id {

}
```
The same goes for class selectors. It would issue a warning for this case, but if you ignored it, this would be valid. However, I'm inclined to consider this a bug fix rather than a breaking change, so I'll drop the version number back down.

Reply:
We should be careful – but not too careful.

Major revs are a pain

Reply:
Done.

Reply:
I meant to link https://github.com/dart-lang/csslib/pull/106 for context.